src/layouts/logged-in.vue

```diff
@@ -1,27 +1,33 @@
 <template>
-  <el-container style="height: 100vh;">
-    <account-chooser v-if="$store.getters.isAuthenticated && $store.getters.selectedAccount === null" />
-    <disabled-account v-else-if="!isSuperAdmin && $store.getters.selectedAccount.status === accountsStatuses.Disabled" />
-    <template v-else>
-      <aside-component>
-        <template #header-extended>
-          <slot name="header-extended" />
-        </template>
+  <div style="display: flex; flex-direction: column; height: 100%">
+    <MBannerMessage
```

Comment:
On utilise plutôt cette syntaxe sur nos apps :

```suggestion
    <m-banner-message
```